docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     build: .
     environment:
       - 'FIRMWARE_VERSION=2021.08.02.0'
+      - 'HELIUM_API_TIMEOUT_SECONDS=5'
```

Comment:
@vpetersson I've been exposing a lot of environment variables in services but haven't added them to docker-compose because they have defaults. But I like Sebastian's decision to add it here too. Self documenting. Do you think it makes sense to add all envvars here moving forward?

Reply:
I feel like this is a great way to expose the environment variables and allow them to be easily viewed/changed from build to build. Usually, when I deal with environment variables I either create a .env file as seen [here](https://docs.docker.com/compose/env-file/) or I include them in the docker-compose file as we are doing so now.